cmd/params/params.go

```diff
@@ -720,6 +720,20 @@ func LoadStatusBarParams(v *viper.Viper) (StatusBar, error) {
 	}, nil
 }
 
+func safeTimeParse(format string, s string) (parsed time.Time, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			if err != nil {
```

Comment:
This is never reached as it's the output variable. Instead we can refactor into this
```suggestion
func safeTimeParse(format string, s string) (parsed time.Time, err error) {
	defer func() {
		if r := recover(); r != nil {
			err = fmt.Errorf("time.Parse panic: %v", r)
		}
	}()

	parsed, err = time.Parse(format, s)
	return
}
```